src/commands/fortune.js

```diff
@@ -0,0 +1,70 @@
+import { exampleCommand, beep, withStar } from "../utilities/string";
+import { sendError } from "../utilities/message";
+
+import { getOneCardData, getMajorCards } from "../services/tarot";
+import fortuneInfo from "../templates/fortuneInfo";
+
+const error = [
+  beep("You may give bubeep a wrong type."),
+  exampleCommand('"tarot normal" or "tarot single"'),
+]
+
+const randomCardNumber = () => Math.floor(Math.random() * 22)
+const randomReverse = () => Math.floor(Math.random() >= 0.5)
+
+const getNotDuplicatedNumberList = (amount) => {
+  const NumberList = []
+  do{
+    const _cardNumber = randomCardNumber()
+    if(!NumberList.includes(_cardNumber))
+      { NumberList.push(_cardNumber) }
+  }while(NumberList.length < amount)
+  return NumberList
+}
+
+const drawManyCard = async (amount) => {
+  const _numberList = getNotDuplicatedNumberList(amount)
+  const _majorCards = await getMajorCards()
+  const _cardDataList = _numberList.map((number)=> {
+    return { card: _majorCards[number] ,isReversed: randomReverse() }})
+  return Object.values(_cardDataList);
```

Comment:
no need to use `Object.values` with array
just returning the`cardDataList` would be enough